tests/unit/neuralnets/NeuralSoftmaxLayer_unittest.cc

```diff
@@ -31,47 +31,46 @@
  * Written (W) 2014 Khaled Nasr
  */
 
-#include <shogun/neuralnets/NeuralSoftmaxLayer.h>
-#include <shogun/neuralnets/NeuralInputLayer.h>
-#include <shogun/lib/SGVector.h>
-#include <shogun/lib/SGMatrix.h>
-#include <shogun/mathematics/Math.h>
-#include <gtest/gtest.h>
+#include "shogun/neuralnets/NeuralSoftmaxLayer.h"
+#include "shogun/lib/SGMatrix.h"
+#include "shogun/lib/SGVector.h"
+#include "shogun/mathematics/Math.h"
+#include "shogun/neuralnets/NeuralInputLayer.h"
+#include "utils/Utils.h"
+#include "gtest/gtest.h"
 
 using namespace shogun;
 
+class NeuralSoftmaxLayerTest : public ::testing::Test
```

Comment:
this fixture is almost identical to the other one. Is there a reason to have this second one?

Reply:
Yep will do so - I'll add a common fixture with the utility functions in `NeuralLayerUtil` right inside the `unit/neuralnets` folder